index.html

```diff
@@ -0,0 +1,470 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Smashing Magazine</title>
+    <link rel="stylesheet" href="./css/styles.css">
+  </head>
+
+  <body class="width-100">
+    <header class="flex space-around">
+      <div class="nav-logo">
+        <img src="images/logo.png" alt="logo of Smashing margazine">
+      </div>
+      <nav class="margin-top1">
+        <a class="textdeco-none white" href="#">Articles</a>
+        <a class="textdeco-none white" href="#">Guides</a>
+        <a class="textdeco-none white" href="#">Books</a>
+        <a class="textdeco-none white" href="#">Events</a>
+        <a class="textdeco-none white" href="#">Membership</a>
+        <a class="textdeco-none white more-button border-6px" href="#">More</a>
+      </nav>
```

Comment:
- [ ] The navigation bar uses `nav` element with `ul`/`li` inside, however, students are allowed to use `div`.

So please apply the ul/li tags for the links to fulfill the requirements.

Reply:
I have added the ul/li to nav

`<nav class="margin-top1">
        <ul>
        <li><a class="textdeco-none white" href="#">Articles</a></li> 
        <li><a class="textdeco-none white" href="#">Guides</a></li>
        <li><a class="textdeco-none white" href="#">Books</a></li>
        <li><a class="textdeco-none white" href="#">Events</a></li>
        <li><a class="textdeco-none white" href="#">Membership</a></li>
        <li><a class="textdeco-none white more-button border-6px" href="#">More</a></li>
      </ul>
      </nav>`